core/crypto/vrf/p256/p256.go

```diff
@@ -243,7 +244,9 @@ func NewVRFSigner(key *ecdsa.PrivateKey) (vrf.PrivateKey, error) {
 	if !curve.IsOnCurve(key.X, key.Y) {
 		return nil, ErrPointNotOnCurve
 	}
-	return &PrivateKey{key}, nil
+	return &PrivateKey{
+		PrivateKey: key,
+	}, nil
```

Comment:
Similarly here, the whole return statement can probably be on one line - the PrivateKey struct is only little!

return &PrivateKey{PrivateKey: key}, nil

is still pretty readable.

Reply:
done